tests/unit/engine.rs

```diff
@@ -3,6 +3,8 @@ mod tests {
     use super::super::*;
     use crate::lists::FilterFormat;
     use crate::resources::MimeType;
+    use base64::prelude::*;
+    use base64::Engine;
 
```

Comment:
1. `base64::Engine` doesn't appear to be used here?
2. instead of importing `base64::prelude::*` in all the files and running into name conflicts with adblock-rust's own structs, let's just selectively import what's actually used? `use base64::prelude::BASE64_STANDARD;` looks like it would be sufficient in all the files.